language/move-prover/boogie-backend/src/prelude/native.bpl

```diff
@@ -62,27 +62,27 @@ function {:inline} $EmptyVec{{S}}(): Vec ({{T}}) {
     EmptyVec()
 }
 
-procedure {:inline 1} $1_Vector_empty{{S}}() returns (v: Vec ({{T}})) {
+procedure {:inline 1} $1_vector_empty{{S}}() returns (v: Vec ({{T}})) {
```

Comment:
This here would be then also fixed. I did it for the 'Extensions' alias but not for the 'Std' alias.